### DISCUSSION TITLE
How can I run your 2.0 WalletConnetSwiftV2 Demo?

### DISCUSSION THREAD

**Discussion by lianghuiboy** [WalletConnect/walletconnect-monorepo#1420] [Developer Support]
When I open the Demo, I need to load the SPM. However, SPM always reports errors and cannot load successfully. Is there any solution?

**Comment by llbartekll** [comment]
hey, maybe, you have some issues with xcode, try:
file->Packages->Reset Package Caches

**Reply by lianghuiboy** [reply]
I need to import it into my project, but after Packages, there is no successful package, so I cannot find the class of WalletConnet

**Reply by llbartekll** [reply]
did you follow the installation guide: https://docs.walletconnect.com/2.0/swift/sign/installation ?
do you `import WalletConnectSign` at the top of your file?

**Reply by lianghuiboy** [reply]
After the fourth step of Tap Add Package, it also failed to load successfully, and then disconnected. Unable to import WalletConnectSign

**Reply by chadyj** [reply]
@bolianghui I'm sorry to hear you are having these issues. Can you share screenshots and walk us through the steps?

**Reply by lianghuiboy** [reply]
https://github.com/WalletConnect/walletconnect-monorepo/discussions/1427

**Reply by llbartekll** [reply]
Ok, I see, so instead of importing `WalletConnectPairing` you need to import `WalletConnectSign`

**Reply by lianghuiboy** [reply]
Sorry, maybe you don't know what I mean. What I want to express is that all files have been imported according to your SPM method. But I couldn't find it
